gazebo_ros/scripts/spawn_entity.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, args):
         parser.add_argument('-wait', type=str, metavar='ENTITY_NAME',
                             help='Wait for entity to exist')
         parser.add_argument('-spawn_service_timeout', type=float, metavar='TIMEOUT',
-                            default=5.0, help='Spawn service wait timeout in seconds')
+                            default=30.0, help='Spawn service wait timeout in seconds')
```

Comment:
Given that this is the same in two places, could it be defined as a constant?

Reply:
Done.